orttraining/orttraining/test/python/orttraining_test_dort_custom_ops.py

```diff
@@ -90,12 +91,22 @@ def create_onnxruntime_session_options():
         session_options.register_custom_ops_library(custom_op_library_path)
         return session_options
 
-    def test_DORT_custom_ops(self):
-        torch._dynamo.reset()
+    def test_export_aten_mul_as_onnx_custom_op_and_run_ort(self):
+        """A Custom Operator Test for DORT
 
-        session_options = TestTorchDynamoOrtCustomOp.create_onnxruntime_session_options()
+        In this test, aten.mul.Tensor is exported to test.customop::CustomOpOne and
+        executed by ORT.
+        """
+        torch._dynamo.reset()
```

Comment:
Looks like reset is done in every test case, maybe move it to setUp()?

Reply:
Will do in another PR. Thanks.